CONTRIBUTING.md

```diff
@@ -105,7 +105,7 @@ If you'd like to test latest Kaoto UI and not rely on a released version, follow
   * `yarn build:lib`
```

Comment:
regarding CI changes, this line needs to be updated too

Reply:
done :+1: